### ISSUE TITLE
1000fps ReHLDS not working

### ISSUE THREAD

**Issue by Feede12** [dreamstalker/rehlds#824]
I have the last update of ReHLDS and the tickrate stay en 500fps. sys_ticrate is 1000 but not respond

My hardware is: Core i3 2120 + 8Gb ram and SSD. Windows server 2019

**Comment by fred0r** [comment]
so after starting the server, when entering `sys_ticrate`via rcon, you see that `sys_ticrate 1000` is set?
are you using any `-pingboost` ?

**Comment by Feede12** [comment]
Yes the sys_ticrate is 1000 but the sv only work on 500. pingboost is a command for linux. i am using windows server

mmtimer show me 1000fps but is erratic and warped

**Comment by fred0r** [comment]
> pingboost is a command for linux. i am using windows server

never really used the windows-version - didnt knew that.

> sys_ticrate is 1000 but the sv only work on 500. 

pls check that `host_frame 0` and `fps_max 0` are set.
maybe try to set `sys_ticrate 2000` and recheck via `stats` if the FPS did change.

**Comment by Feede12** [comment]
thanks for reply

host_framerate 0 and sys_ticrate in 2000 the result is the same:

CPU   In    Out   Uptime  Users   FPS    Players
 0.00  9.95 21.78    2876   611  318.15       4
stats
CPU   In    Out   Uptime  Users   FPS    Players
 1.56 11.28 23.97    2876   611  471.52       4
stats
CPU   In    Out   Uptime  Users   FPS    Players
 0.78 11.52 24.19    2876   611  463.82       4
stats
CPU   In    Out   Uptime  Users   FPS    Players
 0.78 11.31 23.35    2876   611  464.86       4
stats
CPU   In    Out   Uptime  Users   FPS    Players
 0.52 11.39 24.35    2876   611  467.46       4
stats
CPU   In    Out   Uptime  Users   FPS    Players
 0.52 11.59 28.16    2876   611  466.42       4
stats
CPU   In    Out   Uptime  Users   FPS    Players
 0.39 11.49 28.19    2876   611  471.88       4
stats
CPU   In    Out   Uptime  Users   FPS    Players
 0.39 12.95 32.36    2876   611  465.20       4
stats
CPU   In    Out   Uptime  Users   FPS    Players
 0.39 12.62 31.07    2876   611  459.05       4
stats
CPU   In    Out   Uptime  Users   FPS    Players
 0.00 12.12 30.01    2876   611  449.96       4

**Comment by Feede12** [comment]
I installed Ubuntu 16.04 and the fps is the same. Erratic. -pingboost 1 2 3 not working

**Comment by Feede12** [comment]
sys_ticrate 1200 and -pingboost "3" on command line

1000fps stable

**Comment by fred0r** [comment]
Nice - pls close this Issue.
